<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtherChannelsPage</class>
 <widget class="QWidget" name="OtherChannelsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 17px, &quot;SimHei&quot;;	
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;
}</string>
  </property>
  <widget class="QLabel" name="label_emeByPass1">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>259</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_emeByPass2">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>259</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_emeByPass3">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>259</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_ATPCutOff">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>298</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>ATP切除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_car1DoorSafe">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>220</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car1No">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>103</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car2ParkBrake">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>181</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car4No">
   <property name="geometry">
    <rect>
     <x>763</x>
     <y>103</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_emeByPass">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>259</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>紧急制动旁路</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_car1ParkBrake">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>181</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car4BrakeNoRemission">
   <property name="geometry">
    <rect>
     <x>763</x>
     <y>142</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_vehicleNo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>103</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>车号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_car3No">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>103</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car2ATPCutOff">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>298</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car2DoorSafe">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>220</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_emeByPass4">
   <property name="geometry">
    <rect>
     <x>763</x>
     <y>259</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car3DoorSafe">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>220</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car3ATPCutOff">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>298</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car1BrakeNoRemission">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>142</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car2BrakeNoRemission">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>142</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car4DoorSafe">
   <property name="geometry">
    <rect>
     <x>763</x>
     <y>220</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car3ParkBrake">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>181</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car2No">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>103</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car1ATPCutOff">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>298</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_brakeNoRemission">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>142</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>制动不缓解旁路</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_car4ParkBrake">
   <property name="geometry">
    <rect>
     <x>763</x>
     <y>181</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car4ATPCutOff">
   <property name="geometry">
    <rect>
     <x>763</x>
     <y>298</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car3BrakeNoRemission">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>142</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_parkBrake">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>181</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>停放制动旁路</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_doorSafe">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>门安全线旁路</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_EmergencyBroadcast">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>560</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>紧急广播</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
